.github/workflows/integration-test.yml

```diff
@@ -56,7 +58,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
+          python-version: "3.13"
```

Comment:
Not a blocker but, it would be nice if this a matrix of python versions that we run tests on. Also, would be nice if we could have a matrix of Airflow versions to run the tests on. Because that matrix would run in the official Airflow repo. So would be nice if we can catch issues if any early on. Also could be follow-up ed in a separate PR if you'd like to.

Reply:
yes, i had been thinking about that as well. i've added it now in [c30c222](https://github.com/astronomer/airflow-provider-great-expectations/pull/221/commits/c30c222c97607fc2e4e89d98a3e454325b3564ba)